src/_zkapauthorizer/tests/privacypass.py

```diff
@@ -64,10 +64,12 @@ def make_passes(signing_key, for_message, random_tokens):
         for verification_key in verification_keys
     )
     passes = list(
```

Comment:
`passes` is now a list of `bytes` and gets returned but the docstring still says the return type is `list[unicode]`

Reply:
It is actually a list of `Pass`es. I've updated the docstring to match.

Reply:
Actually, this comment (and thus the change I made to the docstring) is incorrect, and i was confusing what changed here with #269.